src/ui/Controls/SubtitleListView.cs

```diff
@@ -367,10 +365,10 @@ public SubtitleListView()
                 ShowGapColumn(LanguageSettings.Current.General.Gap);
             }
 
-            _syntaxColorLineTimer = new Timer { Interval = 41 };
+            _syntaxColorLineTimer = new Timer { Interval = 55 };
             _syntaxColorLineTimer.Tick += SyntaxColorLineTimerTick;
 
-            _setStartAndDurationTimer = new Timer { Interval = 3 };
```

Comment:
Timer component is single-threaded, and is limited to an accuracy of 55 milliseconds

See: https://learn.microsoft.com/en-us/dotnet/api/System.Windows.Forms.Timer?view=netframework-4.8